README.rst

```diff
@@ -0,0 +1,113 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME
+PDFmorph is a Python software package designed to increase the insight
+researchers can get into spectra collected for atomic pair distribution 
+functions.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
```

Comment:
This should most likely use the conda-forge  version

Reply:
Oh man, I just added a bunch about anaconda. For conda-forge would it be the same command except `conda-forge` in place of `conda`? I guess that means some fixes to the previous couple of paragraphs!

Reply:
No you need to change this to
```suggestion
     conda config --add channels conda-forge
```

Reply:
with no other changes necessary?

Reply:
👍